openmmtools/tests/test_utils.py

```diff
@@ -79,7 +79,8 @@ def test_is_quantity_close():
                   (1.01325*unit.bar, 1.01325000006*unit.bar, True),
                   (1.01325*unit.bar, 1.0132500006*unit.bar, False)]
     for quantity1, quantity2, test_result in test_cases:
-        assert is_quantity_close(quantity1, quantity2) is test_result
+        msg = "Test failed: ({}, {}, {})".format(quantity1, quantity2, test_result)
+        assert is_quantity_close(quantity1, quantity2) == test_result, msg
```

Comment:
NICE CATCH! That was driving me crazy.

Reply:
That was @andrrizzi ! He identified this issue in https://github.com/choderalab/openmmtools/pull/320/commits/00d2df1508000f8961ba6026d58a2c710f630532 and pointed me to the fix.